file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,592 @@
+INTERFACE zif_aff_cfdf_v1
+  PUBLIC.
+
+  TYPES: ty_field_name              TYPE c LENGTH 30.
+  TYPES: ty_component_identifier    TYPE c LENGTH 5.
+  TYPES: ty_busienss_context        TYPE c LENGTH 30.
+  TYPES: ty_cds_view_name           TYPE c LENGTH 30.
+  TYPES: ty_badi_implemenation_name TYPE c LENGTH 30.
+  TYPES: ty_code_value              TYPE c LENGTH 30.
+  TYPES: ty_data_subject_id_type    TYPE c LENGTH 10.
+  TYPES: ty_field_type              TYPE c LENGTH 10.
+  TYPES: ty_message_direction       TYPE c LENGTH 3.
+  TYPES: ty_field_control_property  TYPE c LENGTH 1.
+  TYPES: ty_aggregation             TYPE c LENGTH 5.
+
+  TYPES:
+    "! <p class="shorttext">Label and Tooltip</p>
+    "! Label and tooltip
+    BEGIN OF ty_gs_ui_text,
+      "! <p class="shorttext">Translation Language</p>
+      "! Translation language
+      "! $required
+      language      TYPE sy-langu,
+
+      "! <p class="shorttext">Field Label</p>
+      "! Field label
+      "! $required
+      "! $maxLength 40
+      field_label   TYPE c LENGTH 40,
+
+      "! <p class="shorttext">Field Tooltip</p>
+      "! Field tooltip
+      "! $required
+      "! $maxLength 60
+      field_tooltip TYPE c LENGTH 60,
+    END OF ty_gs_ui_text.
+
+  TYPES: ty_gt_ui_text TYPE STANDARD TABLE OF ty_gs_ui_text WITH KEY language.
+
+  TYPES:
+    "! <p class="shorttext">Business Context</p>
+    "! Business context
```

Comment:
For "table entries" we usually add "Details" to the end of the title. 

@albertmink Do you agree? If so, there are more places where "Details" should be added

```suggestion
    "! <p class="shorttext">Business Context Details</p>
    "! Business context details
```

Reply:
If the question is related to UI layout, then there should be 'Business Contexts' and 'Business Context Details'.
Otherwise, I had a look into the JSON Schema and there were deeply nested 'Business Context' within 'Business Context',. That should be un-nested I guess.

Reply:
There are not that many details. It's just the name of the 'Business Context' and an additional detail 'Deviating Field Name'. We adapted the title to 'Business Context Details'...

Reply:
@k-schaser Thanks for adapting. There are some more places where, I think, "... Details" should be added. I will refer to most of the in my upcoming review